mopidy/core/history.py

```diff
@@ -57,3 +57,27 @@ def get_history(self):
         :rtype: list of (timestamp, :class:`mopidy.models.Ref`) tuples
         """
         return copy.copy(self._history)
+
+    def _export_state(self):
+        """Internal method for :class:`mopidy.Core`."""
```

Comment:
The docstring should be removed as it doesn't tell more than the underscore prefix on the method name. Same applies to all the other core controller state management helpers.

Also, the naming lacks a bit of symmetry: It would be more symmetric with 
export/import or store/restore, not export/restore. I'm not suggesting any of these namings, just pointing out the asymmetry to start a discussion about proper names here.